app/dao/notifications_dao.py

```diff
@@ -406,103 +408,22 @@ def _move_notifications_to_notification_history(notification_type, service_id, d
         # This enables us to break this into smaller database queries
         timestamp_to_delete_backwards_from = day_to_delete_backwards_from - timedelta(hours=hour_delta)
 
-        if service_id == '539d63a1-701d-400d-ab11-f3ee2319d4d4':
-            current_app.logger.info(
-                "Beginning insert_update_notification_history for GOV.UK Email from {} backwards".format(
-                    timestamp_to_delete_backwards_from
-                )
-            )
-
-        insert_update_notification_history(notification_type, timestamp_to_delete_backwards_from, service_id, qry_limit)
-
-        if service_id == '539d63a1-701d-400d-ab11-f3ee2319d4d4':
-            current_app.logger.info(
-                "Beginning _delete_notifications for GOV.UK Email {} backwards".format(
-                    timestamp_to_delete_backwards_from
-                )
-            )
-
-        deleted += _delete_notifications(
-            notification_type, timestamp_to_delete_backwards_from, service_id, qry_limit
+        deleted += insert_notification_history_delete_notifications(
+            notification_type=notification_type,
+            service_id=service_id,
+            timestamp_to_delete_backwards_from=timestamp_to_delete_backwards_from,
+            qry_limit=qry_limit
         )
 
-    return deleted
-
-
-def _delete_notifications(notification_type, date_to_delete_from, service_id, query_limit):
-    subquery = db.session.query(
-        Notification.id
-    ).join(NotificationHistory, NotificationHistory.id == Notification.id).filter(
-        Notification.notification_type == notification_type,
-        Notification.service_id == service_id,
-        Notification.created_at < date_to_delete_from,
-    ).limit(query_limit).subquery()
-
-    deleted = _delete_for_query(subquery)
-
-    subquery_for_test_keys = db.session.query(
-        Notification.id
-    ).filter(
+    Notification.query.filter(
         Notification.notification_type == notification_type,
         Notification.service_id == service_id,
-        Notification.created_at < date_to_delete_from,
+        Notification.created_at < day_to_delete_backwards_from,
```

Comment:
Very minor but the delete above for the normal notifications is `<=` and this one for the test ones is just `<`